### ISSUE TITLE
[A11Y] OMP Editorial View | Components Tab in Workflow > Submission > Components tab is not able to be ordered using keyboard

### ISSUE THREAD

**Issue by Tribunal33** [pkp/pkp-lib#12664]
## Issue Description
Functionality not able to be completed.  In general the components area was difficult to navigate.  Once you select Order the button to cancel ordering was on the bottom of the screen.  Mobile view was also noted as missing some buttons and hard to find.  The scope of this ticket will deal with the lack of ordering via keyboard not mobile yet. 
the auditor mentioned that they "could not figure out how to do this." "even tried using the Jaws cursor, and was still unable to do it." 

Lastly, on OMP I can see other areas where this ordering functionality essentially does not work via keyboard.  It expects mouse drag and drop.  `checklists` tab is another area.  When fixing this if the developer associated with the ticket can do a code analysis and highlight the impact areas so they can be tested that would be helpful.  TOtherwise, there maybe some areas that might be missed.  

Also on mobile device it was noted that it couldn't be done either.  Mobile on editorial view should be out of scope though.

## How to reproduce / Current behavior
Navigate to Workflow > Submission > Components tab and using the keyboard only tab and arrow keys try to reorder the list.  I couldn't do this intuitively. 

Additional Impact areas to be covered with these changes.
- [ ] Navigate to Workflow > Submission > Checklists tab 

## What application are you using?
 OMP version 3.5.0-4

## Stack used 
- Device: Laptop
- OS: Windows 11
- Browser: Chrome
- Version: Latest

## Testing method
Manual, Using screenreader and keyboard buttons.

## Action
Etther the enter button or the spacebar should highlight the one that wants to be moved and then arrow keys can highlight another space to move it to by pressing the enter / spacebar buttons again.  

## Additional information

Here is the components tab that is referenced.
<img width="1543" height="507" alt="Image" src="https://github.com/user-attachments/assets/20c180f5-5b19-48c7-8e96-826808305bd6" />
